app/routes/$redirects.tsx

```diff
@@ -0,0 +1,23 @@
+import type {LoaderArgs} from '@remix-run/cloudflare'
+import {redirect} from '@remix-run/cloudflare'
+import {loadRedirects} from '~/server-utils/stampy'
+
+export const loader = async ({request, params}: LoaderArgs) => {
+  try {
+    const {data: redirects} = await loadRedirects(request)
+    const to = params['*'] && redirects[params['*'].replace(/^\/+/, '')]
+    if (to) return redirect(to.replace(/^\/+/, '/'))
```

Comment:
can we move this replace to `loadRedirects`? (so both Coda fields are fixed there and this function will be responsible only for request)